octavia/api/app.py

```diff
@@ -14,6 +14,7 @@
 
 import sys
 
+import rate_limit as ratelimitmiddleware
```

Comment:
It would be wiser to try the import and check later if the import succeeded. This way the test-requirements.txt don't need the rate limit middleware - there is no tests anyway in octavia.